### PR TITLE
Added MetaData fields for new model outputs

### PR DESCRIPTION
**Pull request by mitchellgoffpc**

### CODE REVIEW

File: log.capnp [hunk 1]
```diff
@@ -676,10 +676,17 @@ struct ModelDataV2 {
   struct MetaData {
     engagedProb @0 :Float32;
     desirePrediction @1 :List(Float32);
-    brakeDisengageProb @2 :Float32;
-    gasDisengageProb @3 :Float32;
-    steerOverrideProb @4 :Float32;
+    brakeDisengageProbDEPRECATED @2 :Float32;
```
Comment (adeebshihadeh) [inline log.capnp:679]: move deprecated fields to the bottom

**Comment by HaraldSchafer** [comment]
forwardCollisionDetected implies a collision has been detected, can you change it to something more correct? Maybe "hardBrakePredicted"?

**Comment by adeebshihadeh** [comment]
"forwardCollisionWarning" or "forwardCollisionImminent"?

**Comment by HaraldSchafer** [comment]
Hmm the brake names are not great either. The units are m/s**2. Maybe just call it brake3Probs, brake4Probs, brake5Probs? Or light, medium, hard. Don't really have any good ideas, but the ms is just misleading.

**Comment by HaraldSchafer** [comment]
> "forwardCollisionWarning" or "forwardCollisionImminent"?

@adeebshihadeh The warning should trigger for any needed brake event though, like a red light, a sharp turn. So not necessarily a forward collision.

**Pull request merged**